### ISSUE TITLE
Mate Quest2 Flutter Support

### ISSUE THREAD

**Issue by au-top** [flutter/flutter#101799]
As an android app, fluent to APK can work well on mate quest2

But I still found some problems, and these problems will not appear in Android native applications,

For example, when using the handle to tap and scroll listview, the movement speed is slow

These do not exist in native Android applications

It would be a great thing if it could be solved

thank you

And I found that something like pageview can't support sliding

## Steps to Reproduce

1.  create code   ListView  and  Button  and PageView
2.  use he

<details>
<summary>Code sample</summary>

```dart

ListView.builder(
                  shrinkWrap: true,
                  itemCount: 200,
                  itemBuilder: (context, index) {
                    return SizedBox(child:Text("123"),height:200.0);
                  },
                );

PageView(
        children: [
              Text("1"),
              Text("2"),
        ],
      );
```

</details>

**Comment by darshankawar** [comment]
@au-top 
By Mate Quest2, are you referring to VR headsets ? If so, are you looking to have flutter support for those ?

**Comment by au-top** [comment]
> @au-top By Mate Quest2, are you referring to VR headsets ? If so, are you looking to have flutter support for those ?

yes ! It generally works well, but the controller has a little problem with the click and scroll of the page

**Comment by darshankawar** [comment]
@au-top 
I doubt if Flutter would work properly in VR because I don't think there's any support for it and don't think its feasible for flutter team to support it fully. See [this](https://stackoverflow.com/questions/50645709/does-flutter-support-virtual-or-augmented-reality) link for your reference and check if it helps.

Closing this issue for now, as there's no official support for VR/AR apart from community plugins.

**Comment by au-top** [comment]
> 



> @au-top I doubt if Flutter would work properly in VR because I don't think there's any support for it and don't think its feasible for flutter team to support it fully. See [this](https://stackoverflow.com/questions/50645709/does-flutter-support-virtual-or-augmented-reality) link for your reference and check if it helps.
> 
> Closing this issue for now, as there's no official support for VR/AR apart from community plugins.

sorry 

Maybe there's something wrong with my description


I'll upload a video later to illustrate all this

**Comment by darshankawar** [comment]
ok. Please provide one and I'll reopen this.

**Comment by au-top** [comment]
https://user-images.githubusercontent.com/44134842/163179684-dd9358a5-cc37-445a-ada5-43cf678f8955.mp4

**Comment by au-top** [comment]
> com.oculus.shellenv-20220413-201555.mp4

I opened two apps

Flutter is the first one I developed

The second is that the clash for Android project is a native Android program. I directly used the APK installation package on my mobile phone. It is not an Android program specially adapted to mate Quest 2

**Comment by darshankawar** [comment]
Reopening.

**Comment by darshankawar** [comment]
@au-top 
Is there a way to properly and easily replicate this behavior directly without needing any external tool ? What's the expected result you are looking as compared to current one ?

**Comment by au-top** [comment]
> @au-top Is there a way to properly and easily replicate this behavior directly without needing any external tool ? What's the expected result you are looking as compared to current one ?

I think it's feasible. I think it's about the compatibility between the flutter and the handle

If you can simulate the Android handle operation signal, I think you can show the problem without using the device

I want to scroll listview faster using the Android joystick

And the click or rocker of the handle can better operate pageview

**Comment by darshankawar** [comment]
I am not completely sure, but does this package help ? https://pub.dev/packages/joystick

Also should issue title should still remain same or can you update it to reflect exact requirement ?
